src/extensions/npm/extensions/package.json

```diff
@@ -1,25 +0,0 @@
-{
```

Comment:
We should not remove these.

Reply:
Yes that is true, I am finding that `make dev -B` delete these once webpack starts to be run. Are you seeing this?